pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "hightime"
 description = "Hightime Python API"
 license = "MIT"
 keywords = ["hightime"]
-version = "1.0.1.dev0"
+version = "1.1.0.dev0"
 classifiers = [
     "Development Status :: 5 - Production/Stable",
```

Comment:
Do we want to keep Production/Stable status or go back down to active development status?

Reply:
BTW, Poetry's code for "enriching" the classifiers seems to add the license classifier and Python version classifiers, but not the development status classifier: 
https://github.com/python-poetry/poetry-core/blob/main/src/poetry/core/packages/package.py#L331